sdk/lib/cpprt/terminate.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:     ReactOS c++ runtime library
+ * LICENSE:     FSF Unlimited License (https://spdx.org/licenses/FSFULLR.html)
+ * PURPOSE:     __std_terminate implementation
+ * PROGRAMMER:  Timo Kreuzer (timo.kreuzer@reactos.org)
```

Comment:
Let me be a little annoying here :)
According to our code style, here should be a "COPYRIGHT" line
https://reactos.org/wiki/Coding_Style

Reply:
Fixed